substrate/client/transaction-pool/src/graph/pool.rs

```diff
@@ -358,7 +373,8 @@ impl<B: ChainApi, L: EventHandler<B>> Pool<B, L> {
 			}
 		}
 
-		log::debug!(target: LOG_TARGET,"prune: validated_counter:{validated_counter}, took:{:?}", now.elapsed());
+		let known_txs_count = known_provides_tags.map(|inner| inner.len()).unwrap_or(0);
+		log::debug!(target: LOG_TARGET, validated_counter, known_txs_count, unknown_txs_count, reused_txs_count, duration= now.elapsed(),"prune");
```

Comment:
```suggestion
		log::debug!(target: LOG_TARGET, validated_counter, known_txs_count, unknown_txs_count, reused_txs_count, duration = now.elapsed(),"prune");
```

Reply:
Actually, we need this: https://github.com/paritytech/polkadot-sdk/pull/8001 to be able to use structured logging. `log` crate doesn't support it.